service/resource/deploymentv1/metric.go

```diff
@@ -19,7 +19,3 @@ var versionBundleVersionGauge = prometheus.NewGaugeVec(
 	},
 	[]string{"major", "minor", "patch"},
 )
-
-func init() {
-	prometheus.MustRegister(versionBundleVersionGauge)
-}
```

Comment:
We have to remove the metrics registration in one place because this panics when the same metric is registered twice.